### ISSUE TITLE
Add integration tests for example bot with mad-fake-slack

### ISSUE THREAD

**Issue by CyberLight** [maddevsio/mad-fake-slack#18]
- [x] Add jest + puppeeter to dev container
- [x] Add npm command for running tests
- [x] Write simple scenario for checking different messages which bot will send back to sender in channel and direct 

**Time spent:** 6h 3m `24.06.2019`
**Time spent:** 5h 31m `25.06.2019`

**Comment by CyberLight** [comment]
But problems with parallel on sequential tests running occurred. Need to decide how to run tests without collisions between runs.

**Comment by CyberLight** [comment]
Today I will add Jest environment needed for starting and stopping needed services before/after each test

**Comment by CyberLight** [comment]
* jest-puppeeter problems:
  * Does not properly work killing of process before start
  * If I used `bail: 1` (settings parameter of jest) then global hook for teardown does not called. I removed this usefull setting from config, because of a problem occurred.
* Decide to not use Custom node.js environment, because a lot of work needed for running tests in parallel. Now only one way work properly and it's way to run series of tests wuth `--runInBand` command. Previously in another projects I used this simple method for running integration tests. 

**Result:**
* Now only one instance of mad-fake-slack started on setting up all tests
* Every test run and kill instance of bot
* In memory database of mad-fake-slack is cleared by making `GET` request to `/test/api/db/reset`

**What need to do**
* Add "How to" to README for running example of integration tests with bot and mad-fake-slack
